### DISCUSSION TITLE
A form of global hotkeys is almost necessary to make a splitter usable to most people.

### DISCUSSION THREAD

**Discussion by XeZey** [LiveSplit/LiveSplitOne#916] [General]
It seems like this project has existed for a long time, and it also seems that it is being developed actively still, so I'm curious, are global hotkeys as a standard setting ever coming? There have been methods for getting them to work for quite some time now but they all require extra stuff. My assumption is that most people would consider a splitter that doesn't have global hotkeys to be useless, unless you are doing something weird where you are keeping your phone next to your pc and have it drain its battery the entire time. Also to mention, its possible that its just extremely difficult to implement this, I know next to nothing about programming, but I feel like the importance of this feature being a standard thing is never really mentioned for some reason. There are probably tons of people finding this and then being disappointed once they realize they cant really do anything with it in its default state.

**Comment by CryZe** [comment]
Browsers simply do not provide a way to do this. There's not really anything we can do without you installing something on top, like maybe a browser extension (which maybe is able to do it) or some sort of local application.

**Reply by XeZey** [reply]
would adding a short guide on how to do such in the main github page or preferably even the web page itself be something that could be done? unless you actively search for "livesplitone global hotkeys" and look through the github issues/discussions, you basically dont even know that these workarounds could work or exist, so Im guessing a lot of people wouldnt even bother at that point. I remember spending like 20 minutes on the livesplitone page trying to find a setting for global hotkeys before searching for it online and being confused that the absence of such a feature itself wasn't mentioned anywhere at all.

**Reply by CryZe** [reply]
These workarounds don't exist yet either, so that's something I will look into. I'll try to prioritize this. Also I think you are right, I'll also add a note explaining that global hotkeys are currently not possible.
